CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+Including your operating system and its version, your rust compiler version (`rustc -vV`), and any commands you may have run, as well as their output.
+
+The second category of issue is a bug report.
+Make sure to describe your environment as above, and to include a snippet of code that demonstrates the bug.
+Without a way to reproduce the bug, it will be very hard to fix, so make sure to provide as much information as possible.
+
+The third category of issue is a feature request.
+Quicksilver is an evolving project that can definitely use a lot more features, and the project is very open to suggestions.
+However, there is a certain category of feature that (generally) won't be considered for work: anything that will only work on one of desktop or web.
+A firm goal of quicksilver is unsurprising behavior when taking a web-first project and compiling for desktop and vice-versa.
+Also, there's no guarantee anyone else will be up to working on your feature request; if that's the case, then feel free to [open a PR](#prs).
+
+
+## PRs
+
+The first rule of PRs to Quicksilver is to generally try to make only a few changes in a single PR.
+This way each PR maps to about one commit message, which in turn maps to one set of atomic changes to the codebase.
+(There are exceptions: if you are making a PR to replace a rendering backend, there is going to be a large diff.)
+
+Below is more information about the project organization and styleguides for Quicksilver, which is helpful to read before submitting a change.
+
+### Branch structure
+
+The main branch of quicksilver is `master`, which generally tracks the previous release on crates.io. Changes that are intended for the next version
+are merged into `development`, and then all merged into `master` when it is time for a release.
+
+### Git styleguide
+
+- Use present-tense commands ("do this thing" instead of "does this thing" or "did this thing") so the git log reads like a series of patches
+- Keep line lengths at 72 characters or below (for good commit formatting)
+- The first line of your commit should be a summary
+- Any body text should be seperated from the summary by a blank line
+- When referring to an issue or pull request, use its number so that Github auto-formats it correctly
+
+### Documenting your changes
+
+- Add an entry in `CHANGES.md`, which can generally be the summary(s) of your git commits
+- If you changed the public API, note [BREAKING] in front of your change
+- Additionally, make sure you didn't break the code snippet in the website, the README, or `src/lib.rs`. If you did, make sure to update them.
+
+### Testing your changes
+
+- Make sure to run `cargo test` which will both verify the tests pass and also make sure the crate and all examples compile
+- Make sure to run `cargo +nightly check --target wasm32-unknown --examples` which will verify that the crate and all examples compile for web
```

Comment:
The command produced the following error:
```
error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `rustc - --crate-name ___ --print=file-names --target wasm32-unknown --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro` (exit code: 101)
```
Adding an extra ``-unknown`` made rustc happy:
``cargo +nightly check --target wasm32-unknown-unknown --examples``

Reply:
Yup sorry about that typo! Pushed a fix.